.github/workflows/node-docker.yml

```diff
@@ -79,11 +96,15 @@ jobs:
           username: subquerynetwork
           password: ${{ secrets.SQ_DOCKERHUB_TOKEN }}
 
-      ## node
-      - name: Get updated node version
-        id: get-node-version
+      - run: yarn
+
+      - name: build
+        run: yarn build
+
+      # Replace workspace:* with actual versions
+      - name: Replace workspace dependencies with actual versions
         run: |
-          sh .github/workflows/scripts/nodeVersion.sh
+          sh .github/workflows/scripts/replace-versions.sh
```

Comment:
It would be great if we could run this from the dockerfile instead. That way its really easy to build locally and the changes are contained to the image

Reply:
That was my go to option at first but, we only copy node package into docker, so we cannot access the versions of the other packages in the workspace.

Reply:
I think we could copy in the package.json files (or other packages) in the first step of the docker image

Reply:
I still think we need to do this in the docker image. From a user perspective its going to make building images much easier